RCPTT_Tests/BlocksTests/BlocksAreFormattedCorrectly.test

```diff
@@ -0,0 +1,54 @@
+--- RCPTT testcase ---
+Format-Version: 1.0
+Element-Name: BlocksAreFormattedCorrectly
+Element-Type: testcase
+Element-Version: 3.0
+External-Reference: 
+Id: _Vcj1YFEBEeeEZqjlkcOTbQ
+Runtime-Version: 2.1.0.201606221726
+Save-Time: 6/14/17 3:54 PM
+Testcase-Type: ecl
+
+------=_.content-0a7243a0-75d3-3d5f-9791-539de0e5b7ac
+Content-Type: text/ecl
+Entry-Name: .content
+
+proc configure_simple_ioc [val name] {
+	let [val window [edit_current_config]] {
+ 		add_ioc -ioc_name "SIMPLE" -window $window
+ 		with [get_ioc_dialog] {
+	 		get-button "Auto-Start" | check
+	     	get-combo -after [get-label "Sim. Level"] | select RECSIM
+	     	get-button OK | check_enabled_and_click
+	    }
+ 		add_blocks_to_simple_ioc $window
+ 		save_config
+ 	}
+ }
+ 
+proc add_blocks_to_simple_ioc [val window]{
+ 	add_block "LONG" "SIMPLE:LONG" $window
+ 	add_block "INT" "SIMPLE:VALUE1" $window
+ 	add_block "P3" "SIMPLE:VALUE:P3" $window
+ 	add_block "P5" "SIMPLE:VALUE:P5" $window
+ 	add_block "STRING" "SIMPLE:HELLO" $window
+}
+ 
+
+let [val configname [test_prefix "blocks_config06"]] {
+	create_empty_config -config_name $configname
+ 	load_config -name $configname
+}
+
+configure_simple_ioc
```

Comment:
Why not do this as `with [new_config]` so you don't have two changes to the current configuration. It would keep the test length down a bit.